x-pack/test/reporting/configs/chromium_api.js

```diff
@@ -4,26 +4,32 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { getReportingApiConfig } from './api';
+import { ReportingAPIProvider } from '../services';
 
 export default async function ({ readConfigFile }) {
-
-  const reportingApiConfig = await getReportingApiConfig({ readConfigFile });
+  const apiConfig = await readConfigFile(require.resolve('../../api_integration/config.js'));
+  const functionalConfig = await readConfigFile(require.resolve('../../functional/config.js'));
 
   return {
-    ...reportingApiConfig,
-    junit: {
-      reportName: 'X-Pack Chromium API Reporting Tests',
-    },
+    servers: apiConfig.get('servers'),
+    junit: { reportName: 'X-Pack Chromium API Reporting Tests' },
     testFiles: [require.resolve('../api/chromium_tests')],
+    services: {
+      ...apiConfig.get('services'),
+      reportingAPI: ReportingAPIProvider,
+    },
     kbnTestServer: {
-      ...reportingApiConfig.kbnTestServer,
+      ...apiConfig.get('kbnTestServer'),
       serverArgs: [
-        ...reportingApiConfig.kbnTestServer.serverArgs,
+        ...functionalConfig.get('kbnTestServer.serverArgs'), // optimizer config
```

Comment:
I think I might add a longer comment, e.g. 

```
// Use functionalConfig instead of apiConfig because we want all the UI settings, as reporting requires
// them, and, for example, can't have optimize setting enabled.
```

Reply:
e3659a5f27